### ISSUE TITLE
SDK33 ejected with expokit, android sync failed with 'no value for <appAuthRedirectScheme> is provided'

### ISSUE THREAD

**Issue by darkxinyu** [expo/expo#4518]
### Environment

  Expo CLI 2.19.4 environment info:
    System:
      OS: macOS 10.14.5
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 10.15.3 - ~/.nvm/versions/node/v10.15.3/bin/node
      Yarn: 1.16.0 - /usr/local/bin/yarn
      npm: 6.9.0 - ~/.nvm/versions/node/v10.15.3/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Android Studio: 3.4 AI-183.6156.11.34.5522156
      Xcode: 10.2.1/10E1001 - /usr/bin/xcodebuild
    npmPackages:
      @types/react: ^16.8.19 => 16.8.19
      @types/react-native: ^0.57.60 => 0.57.60
      expo: ^33.0.0 => 33.0.4
      react: 16.8.3 => 16.8.3
      react-native: https://github.com/expo/react-native/archive/sdk-33.0.0.tar.gz => 0.59.8
    npmGlobalPackages:
      expo-cli: 2.19.4

app's target: ExpoKit (for Android)

### Steps to Reproduce

1、expo init project
2、expo eject --eject-method expokit
3、open ejected android project in Android Studio

### Expected Behavior

successfully synced

### Actual Behavior

sync failed for the following reason:

  Execution failed for task ':expo-app-auth:processDebugAndroidTestManifest'.
    Manifest merger failed : Attribute data@scheme at manifestMerger2378475533296953441.xml requires a placeholder substitution but no value for <appAuthRedirectScheme> is provided.

### temporary workaround

edit `node_modules/expo-app-auth/android/build.gradle` and add 
```
manifestPlaceholders = [
    'appAuthRedirectScheme': 'host.exp.exponent'
]
```
to `android.defaultConfig`

then the project synced without error, but we consider this a bad practice to have to modify a library file

**Comment by sjchmiela** [comment]
That's interesting, could you check if you have

https://github.com/expo/expo/blob/9f099b0e2661ef1dc52d43d9fb86618f2a665ace/android/app/build.gradle#L45-L48

fragment in your `app/build.gradle`?

**Comment by darkxinyu** [comment]
Yes, it's there, and I found a issue in another repo that talks something about 
https://github.com/openid/AppAuth-Android/pull/341

**Comment by sjchmiela** [comment]
Interesting, what Gradle versions do you use? Go into `android` folder and run `./gradlew version` or `./gradlew --version`.

**Comment by darkxinyu** [comment]
------------------------------------------------------------
Gradle 4.10.2
------------------------------------------------------------

Build time:   2018-09-19 18:10:15 UTC
Revision:     b4d8d5d170bb4ba516e88d7fe5647e2323d791dd

Kotlin DSL:   1.0-rc-6
Kotlin:       1.2.61
Groovy:       2.4.15
Ant:          Apache Ant(TM) version 1.9.11 compiled on March 23 2018
JVM:          1.8.0_151 (Oracle Corporation 25.151-b12)
OS:           Mac OS X 10.14.5 x86_64

**Comment by sjchmiela** [comment]
Hm, I have my machine setup exactly like you and I haven't encountered an error like you did. 🤔

**Comment by ricardomoura** [comment]
having exactly the same issue. same gradle version

**Comment by darkxinyu** [comment]
Unfortunately, the issue still exists even after a `$ ./gradlew clean` and `Invalidate and Restart` of Android Studio ..

**Comment by c00piser** [comment]
I was having the same problem after upgrade Android Studio to ver 3.4. Then I downgrade back to 3.3.2 and the problem solved

**Comment by ShimShamSam** [comment]
Same thing as  @c00piser - I updated to Android Studio 3.4 and started receiving this error. Downgrading to 3.3.x got rid of it.

**Comment by WouterFlorijn** [comment]
Can we get any updates on this from the Expo team? This is still a huge issue with no reasonable fix...

**Comment by sjchmiela** [comment]
Sorry, we haven't had looked at support for Android Studio 3.4+ to our projects yet. In the meantime, downgrading to 3.3 should do the work.

**Comment by onka13** [comment]
@WouterFlorijn  Try "gradlew assembleDebug --stacktrace" command. And check errors in detail. I think this is an IDE problem. When I executed gradlew command, I released that actually I've diff. error about "androidx"

**Comment by pencilcheck** [comment]
Is this still an issue? I'm using Android Studio 3.5 and seems to be getting the same error.

**Comment by niv-breez** [comment]
I got the same error

**Comment by sjchmiela** [comment]
This has been fixed in https://github.com/expo/expo/commit/e22d13b677038549a711c5ba483069043c499c06#diff-bfc4094bc2a61797ab31b207d971acce and the fix has been released in SDK35 (so `expo-app-auth@7.0.0`).
